src/Drupal/DrupalExtension/MinkAwareTrait.php

```diff
@@ -169,4 +169,26 @@ public function saveScreenshot($filename = null, $filepath = null)
         $filepath = $filepath ? $filepath : (ini_get('upload_tmp_dir') ? ini_get('upload_tmp_dir') : sys_get_temp_dir());
         file_put_contents($filepath . '/' . $filename, $this->getSession()->getScreenshot());
     }
+
+  /**
+   * Return a region from the current page.
+   *
+   * @throws \Exception
+   *   If region cannot be found.
+   *
+   * @param string $region
+   *   The machine name of the region to return.
+   *
+   * @return \Behat\Mink\Element\NodeElement
+   */
+  public function getRegion($region)
+  {
```

Comment:
@kumikoono Should we probably follow Drupal coding standard rule at least for our code that we are adding?

```
  public function getRegion($region) {
```

Reply:
Done